.reaction/jsdoc/templates/static/README.md

```diff
@@ -1,32 +1,32 @@
-# Reaction Commerce API Docs
+# Mailchimp Open Commerce API Docs
 
 ## API Docs
 
 Start here:
 
-- [Reaction Components API](Components.html)
-- [Reaction Core schemas](Schemas.html)
+- [MOC Components API](Components.html)
+- [MOC Core schemas](Schemas.html)
```

Comment:
Should we use Mailchimp Open Commerce here?

Reply:
I'm fine with that - I used the acronym to keep it short, but I'm fine with Mailchimp Open Commerce

Reply:
Yeah, the convention I am following is to use Mailchimp Open Commerce everywhere except code, where I use `moc`